rdm/0004-browser-support.md

```diff
@@ -0,0 +1,98 @@
+- Start Date: 2019-10-02
+- RFC PR: [5](https://github.com/inveniosoftware/rfcs/pull/5)
+- Authors: Guillaume Viger
+
+# Support Evergreen Browsers
+
+## Summary
+
+InvenioRDM commits to supporting secure, evergreen browsers that are themselves
+supported by their vendors.
+
+## Terminology
+
+Evergreen Browser: A browser that keeps itself up-to-date without prompting the user.
+
+## Motivation
+
+- As an InvenioRDM core or extension/plugin developer, I want to take advantage
+of modern Web APIs such as flexbox, websockets, File API, modern Javascript (ES2015+)...
+so that I can provide more powerful features at reduced development and maintenance
+(polyfills...) cost.
+
+- As an InvenioRDM hoster, running an application that requires evergreen secure
+browsers benefits me since it ensures my users are running secure browsers
+which reduces my surface of attack and reduces my support costs because evergreen
+browsers continually fix their bugs and security holes.
+
+
+## Detailed design
+
+Microsoft Edge, Google Chrome, Mozilla Firefox are
+supported. Browsers from derivatives should work by consequence.
+
+Apple Safari is updated with Apple's OS'es. The latest available major
+version along with previous 2 major versions should be supported. For example,
+if Safari 12 is the currently available version, then Safari 11 and 10 are
+supported.
+
+To account for mobile browsers that sometimes lag behind their desktop
+equivalent and to prevent usage of bleeding edge features that are
+not widely-adopted, the Edge, Chrome and Firefox user base can be assumed to
+be at least running the fifth browser version prior to the current one. For
+example, if Chrome 80 is the current version, then features in Chrome 75 that
+are still around are fair-game. If there is a known bug in Chrome 76 fixed in
+Chrome 77, a fix should still be provided (although time heals all wounds clearly).
```

Comment:
Reality and experience from production instances would be helpful here to define good support window sizes. These are just first stab (unintentionally ends up with Safari getting much more backwards compatibility I believe)